synapse/tests/test_model_infotech.py

```diff
@@ -997,14 +980,14 @@ async def test_it_forms_hardware(self):
                     :type=pc.laptop
                     :desc=WootWoot
                     :released=20220202
-                    :cpe=cpe:2.3:h:dell:xps13::::::::
+                    :cpe=cpe:2.3:h:dell:xps13:*:*:*:*:*:*:*:*
```

Comment:
For reference I cannot find any captured test data which uses that URI pattern `::`  with a 2.3 string :+1: